server/routes/index.ts

```diff
@@ -14,11 +14,7 @@ export default function indexRouter(router: Router, services: Services): Router
 
   get('/', (req, res) => res.redirect('/calendar#today'))
   get('/contact-us', (req, res) => {
-    res.render('pages/contact-us', {
-      employeeName: req.user?.employeeName,
-      csrfToken: res.locals.csrfToken,
-      authUrl: req.authUrl,
```

Comment:
we've now got a redirect from check my diary so don't need this passing through either